build.gradle

```diff
@@ -1,12 +1,24 @@
 import java.text.SimpleDateFormat
 
+buildscript {
```

Comment:
the whole block of "buildscript" is outdated I think in new gradle version (or we need it at least not more)